README.md

```diff
@@ -1,27 +1,22 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet) ![](https://img.shields.io/badge/-HTML-orange) ![](https://img.shields.io/badge/-CSS-blue) ![](https://img.shields.io/badge/-JavaScript-yellow) ![](https://img.shields.io/badge/-webpack-ABC9FF)
 
-# Awesome Books
-> A basic app project built with HTML, CSS and JS to keep track of awesome books.
-
+# To-Do List 
+> "To-do list" is an app that helps to organize your day. It simply lists the things that you need to do and allows you to mark them as complete.
 
 ## Built With
 
 - **HTML/CSS & JS best practices:** Correct use of tags, elements, properties and syntax.
 - **GitHub flow:**  Correct use of Branches for deployment and features development.
 - **Linters Check:** Local and Pull Request check for errors, bugs and stylistic errors in code.
+- **Webpack:** used as a bundle to compile JavaScript modules.
+
 
 ## Live Demo :arrow_forward:
 
-[Awesome Books SPA](https://raminka13.github.io/BooklistApp/)
+[To-Do List App](https://karlavdelgadof.github.io/ToDo-list-app/)
 
 ## Authors
```

Comment:
##  [OPTIONAL] 
* I strongly suggest you update your readme file with setup instructions on how to successfully install and run your app so that visitors would find it easy to use 👍